woocommerce/woocommerce-android
# Select the first order by default in tablet

Closes: #10569 


#### Note: I suggest to just review just the code and test the functionality of this PR in #10643 as it handles few edge cases.

### Description

This PR selects and opens the first order from the order list by default in a 2 pane layout.

### Testing instructions

1. Open Order listing screen in tablet device
2. Ensure the first order is selected by default and is opened in the detail pane

### Images/gif

[open_first_order.webm](https://github.com/woocommerce/woocommerce-android/assets/1331230/4f30205f-165d-4bc5-8d93-3870625c419c)
